tests/ui/identity_op.rs

```diff
@@ -66,4 +66,10 @@ fn main() {
     let b = a << 0; // no error: non-integer
 
     1 * Meter; // no error: non-integer
+
+    2 % 3;
+    -2 % 3;
```

Comment:
Can the tests be added like this?
```
2 % -3 + x;
-2 % -3 + x;
```

Reply:
Thanks, I updated the tests and fixed to code to check less than on the absolute values of both sides.